homeassistant/components/discovery/__init__.py

```diff
@@ -68,6 +69,7 @@
     SERVICE_HASSIO: ('hassio', None),
     SERVICE_AXIS: ('axis', None),
     SERVICE_APPLE_TV: ('apple_tv', None),
+    SERVICE_ENIGMA2: ('enigma2', None),
```

Comment:
Inserting media_player platform as first item in the tuple would load that platform directly instead of loading the component and firing the event.